db/utility.js

```diff
@@ -21,6 +21,7 @@ const handleComment = (itemTransResult, itemCommentResult, commentResult)=>{
             }
         })
     })
+    // console.log("itemCommentObj", itemCommentObj)
```

Comment:
same for this commented code, if you don't want the actual console.log just delete it